pkgs/by-name/se/sequoia-sq/package.nix

```diff
@@ -74,10 +70,10 @@ rustPlatform.buildRustPackage rec {
   passthru.updateScript = nix-update-script { };
 
   meta = {
-    description = "Cool new OpenPGP implementation";
+    description = "A command line application exposing a useful set of OpenPGP functionality for common tasks.";
     homepage = "https://sequoia-pgp.org/";
     changelog = "https://gitlab.com/sequoia-pgp/sequoia-sq/-/blob/v${version}/NEWS";
-    license = lib.licenses.gpl2Plus;
+    license = lib.licenses.gpl2Only;
```

Comment:
Did they change license from Plus to Only?

https://gitlab.com/sequoia-pgp/sequoia-sq/-/commit/dcea72208a095fd7b99faf365a3e55e2b68f72cf

Reply:
> Did they change license from Plus to Only?
> 
> [gitlab.com/sequoia-pgp/sequoia-sq/-/commit/dcea72208a095fd7b99faf365a3e55e2b68f72cf](https://gitlab.com/sequoia-pgp/sequoia-sq/-/commit/dcea72208a095fd7b99faf365a3e55e2b68f72cf)

I think the license should be changed to `lib.licenses.lgpl2Plus`.